build.gradle

```diff
@@ -34,8 +30,26 @@ allprojects {
 }
 
 subprojects {
-  if (it.name != platformProject) {
+
+  if (it.name == platformProject) {
+    apply plugin: "java-platform"
+  } else {
     apply plugin: "java-library"
+  }
+
+  // Constraints for platform and all other projects
+  dependencies {
+    constraints {
+      api("com.mx.path-core:platform") {
+        version {
+          strictly "[1.4.0,2.0["
```

Comment:
We talked about standardizing on [1.4.0,2.0).

Also, instead of a 'reject "1.4.0"' we could just do (1.4.0,2.0)

Reply:
I'm rejecting it explicitly for clarity

Reply:
This is also a bit experimental to see if this pattern will work for other projects. Right now, the constraints used by the projects and the sister platform project are defined separately, which seems gross.

Reply:
👍